doc/source/notebooks/basics/monitoring.pct.py

```diff
@@ -128,6 +128,21 @@ def plot_prediction(fig, ax):
     opt.minimize(training_loss, model.trainable_variables)
     monitor(step)  # <-- run the monitoring
 
+# %% [markdown]
+# ### Scipy Optimization monitoring
+#
+# Note that if you want to use the scipy optimizer, and want to monitor the training progress, then you need to do
+# something slightly different.
+#
+# ```
+# # define fast_tasks, slow_tasks, training_loss as in done in the previous 2 cells. Then,
+# from gpflow.monitor import ScipyMonitor
+# monitor = ScipyMonitor(fast_tasks, slow_tasks)
+#
+# opt = gpflow.optimizers.Scipy()
+# opt.minimize(training_loss, model.trainable_variables, step_callback=monitor)
+# ```
```

Comment:
Can you turn this into an actual code cell ? That would ensure the code is actually run in the continuous integration and give us more confidence it is (and will stay) correct.